cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -97,7 +97,7 @@ pub type LocationToAccountId = (
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
 	// Foreign locations alias into accounts according to a hash of their standard description.
-	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
+	HashedDescription<AccountId, (DescribeTerminus, DescribeFamily<DescribeAllTerminal>)>,
```

Comment:
Why is this needed? To be able to convert local Root to an account id? What is the actual use case for that?

Reply:
> To be able to convert local Root to an account id?

Exactly. This is to support the `RootLocation` [here](https://github.com/yrong/polkadot-sdk/blob/629386ff5e1c5cb89b7666b221e34f7c0deb2aba/cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs#L189), which allow sudo to register any asset.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/8725/commits/029da092d46c2f4d3ba0ac5d685391dc4c053ffe